src/layout/CheckButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Flex } from './Flex'
+
+let CheckboxDefault = (props) => <input type="checkbox" disabled {...props} />
+
+let CheckButton = ({ Button = 'button', Checkbox = CheckboxDefault, checked = false, onClick, children, ...props }) => (
```

Comment:
CheckButton can't check itself. This is intentional - but, given that, maybe it should take an `isChecked` lens instead? 🤔